lib/galaxy/tool_util/deps/container_resolvers/test.py

```diff
@@ -0,0 +1,29 @@
+from typing import (
+    Container,
+    Optional,
+    TYPE_CHECKING,
+)
+
+from . import ContainerResolver
+
+if TYPE_CHECKING:
+    from ..dependencies import ToolInfo
+    from ..requirements import ContainerDescription
+
+
+class NullContainerResolver(ContainerResolver):
```

Comment:
This seems currently unused?

Reply:
Hmm. If I grep in my current source tree I find it used in `lib/galaxy/app_unittest_utils/galaxy_mock.py`. Could you check the diff with f9116e9308?

Reply:
Wrong commit. It's: 894cdece7a